plugin/mutate/source/dextool/plugin/mutate/backend/report/html/package.d

```diff
@@ -632,6 +633,61 @@ string toVisible(MetaSpan.StatusColor s) {
     return format("status_%s", s);
 }
 
+/// DB data for coverage-visualization
+struct CoverageData {
```

Comment:
Now when it is refactored this much and I can see the impl more clearly I think one last refactoring is appropriate.
The struct is unnecessary. it can be replaced by a "simple" function that returns a `bool[uint]`.

```d
bool[uint] extractLineCovData(CovRegionStatus[] dbData, ref FileCtx ctx) {
  bool[uint] lineList;
  ...
  return lineList;
}
```